SU2_CFD/include/iteration_structure.hpp

```diff
@@ -0,0 +1,1807 @@
+﻿/*!
+ * \file iteration_structure.hpp
+ * \brief Headers of the iteration classes used by SU2_CFD.
```

Comment:
This and the associated .cpp were split into separate files. You will need to delete and commit.